.github/workflows/build.yml

```diff
@@ -61,46 +83,29 @@ jobs:
 
     - name: Install App Store Connect API key
       run: |
-        echo -n "${{ secrets.APP_STORE_CONNECT_API_KEY_BASE64 }}" | base64 --decode -o "${{ runner.temp }}/AuthKey.p8"
-
-    - name: Run tests
-      env:
-        PLATFORM: 'iOS Simulator'
-        DEVICE: 'iPhone 16'
-      run: |
-        set -o pipefail && xcodebuild test -project "${{ env.XCODE_PROJECT }}" \
-          -scheme "${{ env.SCHEME }}" \
-          -testPlan "${{ env.TEST_PLAN }}" \
-          -destination "platform=$PLATFORM,name=$DEVICE" \
-          -resultBundlePath "${{ env.TEST_RESULT_BUNDLE }}" \
-          -clonedSourcePackagesDirPath "${{ runner.temp }}/SourcePackages" \
-          -disableAutomaticPackageResolution | xcpretty
-
-    - name: Upload test result bundle
-      uses: actions/upload-artifact@v4
-      if: ${{ failure() }}
-      with:
-        name: ${{ env.SCHEME }}-${{ github.sha }}.xcresult
-        path: ${{ env.TEST_RESULT_BUNDLE }}
```

Comment:
Removed testing step from the build workflow, because tests are already run on each PR and on each commit on main